lcls-twincat-motion/Library/POUs/Motion/Utils/FB_Read70x1CoeData.TcPOU

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="FB_Read70x1CoeData" Id="{3d4514dc-8223-47db-9879-899720aca4f2}" SpecialFunc="None">
+    <Declaration><![CDATA[FUNCTION_BLOCK FB_Read70x1CoeData
```

Comment:
Did a different wip branch get included in this commit accidentally? There are 2 files added that include CoE utilities.

Reply:
interesting, let me check.